src/palindrom.js

```diff
@@ -475,14 +475,15 @@ const Palindrom = (() => {
             this.remoteUrl = new URL(remoteUrl, this.remoteUrl.href);
         }
 
-        handleResponseHeader(res) {
+        handleSuccessResponse(res, callback) {
             /* Axios always returns lowercase headers */
             const location =
                 res.headers &&
                 (res.headers['x-location'] || res.headers['location']);
             if (location) {
                 this.setRemoteUrl(location);
             }
+            callback && callback.call(this, res, res.config.method.toUpperCase());
```

Comment:
This `res.config.method.toUpperCase()` change is not documented in the commit message. Is this on purpose?

Reply:
Yes. It's just an alternative to passing `method` param around. I thought I needn't to do that since it's inside `res` anyways. This is the param I mean https://github.com/Palindrom/Palindrom/blob/b897d07c62e96c1a6217347efeea59b8254a6770/src/palindrom.js#L498